src/Ext/Techno/Hooks.cpp

```diff
@@ -471,3 +472,59 @@ DEFINE_HOOK(0x4CD4E1, FlyLocomotionClass_Update_LayerUpdate, 0x6)
 
 	return 0;
 }
+
+DEFINE_HOOK(0x7037F7, TechnoClass_Cloak_CloakAnim, 0x5)
+{
+	GET(TechnoClass* const, pThis, ESI);
+
+	if (const auto pAnimType = TechnoTypeExt::ExtMap.Find(pThis->GetTechnoType())->CloakAnim.Get(RulesExt::Global()->CloakAnim))
+	{
+		if (const auto pAnim = GameCreate<AnimClass>(pAnimType, pThis->GetCoords()))
+			AnimExt::SetAnimOwnerHouseKind(pAnim, pThis->Owner, nullptr, false);
+	}
+
+	return 0;
+}
+
+DEFINE_HOOK(0x70374F, TechnoClass_Uncloak_DecloakAnim, 0x5)
+{
+	TechnoClass* const pThis = reinterpret_cast<TechnoClass*>(R->ESI<int>() - 0x9C);
+
+	if (const auto pAnimType = TechnoTypeExt::ExtMap.Find(pThis->GetTechnoType())->DecloakAnim.Get(RulesExt::Global()->DecloakAnim))
+	{
+		if (const auto pAnim = GameCreate<AnimClass>(pAnimType, pThis->GetCoords()))
+			AnimExt::SetAnimOwnerHouseKind(pAnim, pThis->Owner, nullptr, false);
+	}
+
+	return 0;
+}
+
+DEFINE_HOOK(0x4D9992, FootClass_PointerGotInvalid_Parasite, 0x7)
+{
+	enum { SkipGameCode = 0x4D99D3 };
+
+	GET(FootClass*, pThis, ESI);
+	GET(AbstractClass*, pAbstract, EDI);
+	GET(FootClass*, pParasiteOwner, EAX);
+	GET(bool, bRemoved, EBX);
```

Comment:
```suggestion
	GET(bool, removed, EBX);
```

Do not use Hungarian notation for anything other than a pointer.